CLAUDE.md

```
@@ -0,0 +1,103 @@
+
+## 🛠 Common Commands
+
+### Build & Run
+bash
+# 빌드 (테스트 제외)
+./gradlew build -x test
+
+# 로컬 실행
+./gradlew bootRun
+
+### Testing
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)